### ISSUE TITLE
Calendar: PUT "/randomAlphanumChars.ics" and then try to GET a "/randomAlphanumChars" (without .ics)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#335]
```
What steps will reproduce the problem?
1. Create an event. It will PUT a "event.ics" file on the server
2. Just after that, the plugin try to GET "event" file.

What is the expected output? What do you see instead?
Output seems fine, but i don't like seeing this in my logfile.

What version of the product are you using? On what operating system?
- myroundcube SVN Revision: 87
- Roundcube 0.7.2
- Debian Squeeze

Here is the HTTP stream between myroundcube and the caldav server :

PUT 
/groupwareserver.php/calendars/sebastien@twolife.org/default/20120507T131144Z-m5
ap4PW.ics HTTP/1.1
Authorization: Basic ****
Host: 127.0.0.1:4433
Accept: */*
Content-type: text/calendar
Content-Length: 313

BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//Roundcube Webmail//NONSGML Calendar//EN
X-WR-Timezone: Europe/London
BEGIN:VEVENT
DTSTART:20120507T111500Z
DTEND:20120507T123000Z
DTSTAMP:20120507T111144Z
SEQUENCE:1336389104
SUMMARY:test1
DESCRIPTION:rc1
CATEGORIES:Aucun
UID:20120507T131144Z-m5ap4PW
END:VEVENT
END:VCALENDAR

HTTP/1.1 201 Created
Date: Mon, 07 May 2012 11:11:44 GMT
Server: Apache/2.2.16 (Debian)
X-Powered-By: PHP/5.3.3-7+squeeze8
Content-Length: 0
Vary: Accept-Encoding
Content-Type: text/html


GET 
/groupwareserver.php/calendars/sebastien@twolife.org/default/20120507T131144Z-m5
ap4PW HTTP/1.1
Authorization: Basic ****
Host: 127.0.0.1:4433
Accept: */*
Content-Length: 0
Content-Type: application/x-www-form-urlencoded

HTTP/1.1 404 Not Found
Date: Mon, 07 May 2012 11:11:44 GMT
Server: Apache/2.2.16 (Debian)
X-Powered-By: PHP/5.3.3-7+squeeze8
Content-Length: 266
Content-Type: application/xml; charset=utf-8

<?xml version="1.0" encoding="utf-8"?>
<d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
  <s:exception>Sabre_DAV_Exception_NotFound</s:exception>
  <s:message>Calendar object not found</s:message>
  <s:sabredav-version>1.6.2</s:sabredav-version>
</d:error>


```

Original issue reported on code.google.com by `twol...@gmail.com` on 7 May 2012 at 11:33

**Comment by GoogleCodeExporter** [comment]
```
Please test ...

caldav.php ::: line 155

$ret = $this->caldav->GetEntryByHref($caldav_props[0] . '.ics');

Regarding you ticket @ SabreDAV Repo 
(http://code.google.com/p/sabredav/issues/detail?id=214#makechanges):

Please test ...

caldav.php ::: insert after line 108:

$this->caldav->SetDepth(1);

Please report results asap!
```

Original comment by `rol...@liebl.ath.cx` on 8 May 2012 at 6:45
- Changed state: **Reproducing**

**Comment by GoogleCodeExporter** [comment]
```
* caldav.php ::: line 155
-- it works fine, thanks :-)

* caldav.php ::: insert after line 108:
-- not working; i don't see an http header 'Depth' when i look the http stream

```

Original comment by `twol...@gmail.com` on 8 May 2012 at 7:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 9 May 2012 at 4:06
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in calendar v8.8.
```

Original comment by `rol...@liebl.ath.cx` on 10 May 2012 at 7:53
- Changed state: **Fixed**
